internal/managedworker/worker.go

```
@@ -75,6 +75,11 @@ func (w *Worker) Run(ctx context.Context) error {
 			if ctx.Err() != nil {
 				return nil
 			}
+			var responseErr *ResponseError
+			if errors.As(err, &responseErr) && responseErr.Status == 409 {
+				fmt.Fprintf(w.stderr, "machinist: report run %s: %v\n", run.ID, err)
```

golangci-lint (2.13.2)

[error] Error return value of `fmt.Fprintf` is not checked

(errcheck)